dropwizard-migrations/src/test/java/io/dropwizard/migrations/DbDumpCommandTest.java

```diff
@@ -43,7 +43,7 @@
     private TestMigrationConfiguration existedDbConf;
 
     @BeforeClass
-    public static void initXmlParser() throws Exception {
+    public static synchronized void initXmlParser() throws Exception {
```

Comment:
I'm quite sure this is a false positive. `@BeforeClass` methods will be invoked only once per class, this is guaranteed by JUnit. I would just moved the variable assignments from this method directly to theirs declaration.